geode-web/src/integrationTest/java/org/apache/geode/management/internal/cli/commands/CommandOverHttpTest.java

```diff
@@ -89,4 +93,15 @@ public void exportConfig() throws Exception {
         .containsOutput("File saved to " + Paths.get(dir, "server-cache.xml").toString())
         .containsOutput("File saved to " + Paths.get(dir, "server-gf.properties").toString());
   }
+
+  @Test
+  public void commandEncodeDecodeProperly() throws Exception {
+    Region<Object, Object> testRegion =
+        server.getCache().createRegionFactory(RegionShortcut.REPLICATE).create("test");
+    // this command would not fail because of parameter url encoding/decoding
```

Comment:
Don't really need the comment here. Name of test should describe what the test is doing. Consider making test method name more descriptive ie. `command_encodes_and_decodes_special_characters_properly`

Reply:
good suggestion. Will keep that in mind when writing future tests.